subprojects/gdk_rust/gdk_electrum/src/lib.rs

```diff
@@ -287,11 +286,10 @@ fn make_txlist_item(tx: &TransactionMeta) -> TxListItem {
         type_,
         memo: tx.create_transaction.as_ref().and_then(|c| c.memo.clone()).unwrap_or("".to_string()),
         txhash: tx.txid.clone(),
-        transaction_size: len,
         transaction: tx.hex.clone(), // FIXME
         satoshi: tx.satoshi.clone(),
         rbf_optin: tx.rbf_optin, // TODO: TransactionMeta -> TxListItem rbf_optin
-        cap_cpfp: false,         // TODO: TransactionMeta -> TxListItem cap_cpfp
+        can_cpfp: false,         // TODO: TransactionMeta -> TxListItem can_cpfp
         can_rbf: false,          // TODO: TransactionMeta -> TxListItem can_rbf
```

Comment:
This should be true if tx is not confirmed and tx.rbf_optin

Reply:
Addressed in https://github.com/Blockstream/gdk/pull/119/commits/b0796b44e05c7f30514e6390a50fa7c2c150dfcd.